src/lib/journey/config.store.ts

```diff
@@ -43,27 +43,25 @@ const defaultJourneys = {
 // Ensure default follows schema
 journeyConfigSchema.parse(defaultJourneys);
 
-export let configuredJourneys: Readable<StoreItem[]>;
+export let configuredJourneysStore: Writable<StoreItem[]> = writable(
+  (Object.keys(defaultJourneys) as JourneyKeys[]).map((key) => ({
+    ...defaultJourneys[key],
+    key,
+  })),
+);
 
 export function initialize(customJourneys?: z.infer<typeof journeyConfigSchema> | null) {
   if (customJourneys) {
     // Provide developer feedback if customized
     journeyConfigSchema.parse(customJourneys);
 
     const arr = Object.keys(customJourneys) as JourneyKeys[];
-    configuredJourneys = readable(
+    configuredJourneysStore = writable(
```

Comment:
Just saw this. This should be converted into a `set` on the store, rather than recreating the store.